webpack/scenes/Hosts/ChangeContentSource/index.js

```diff
@@ -45,18 +44,18 @@ const ChangeContentSourcePage = () => {
 
   const contentHosts = useSelector(selectContentHosts);
   const hostsWithoutContent = useSelector(selectContentHostsWithoutContent);
-  const environments = useSelector(selectEnvironments);
   const contentSources = useSelector(selectContentSources);
   const jobInvocationPath = useSelector(selectJobInvocationPath);
 
   const template = useSelector(selectTemplate);
   const contentViews = useSelector(selectContentViews);
 
   const [contentSourceId, setCapsuleId] = useState('');
```

Comment:
When I only have a single content source, it's confusing that I have to click it before the environments component becomes enabled. Can this default to the host's current content source?

Reply:
Updated to use a placeholder text for both bulk and single change forms.